build-parsers.sh

```diff
@@ -28,5 +28,7 @@ fi
 
 #Generate the haxe and hxml parsers
 java -jar grammar-kit.jar gen ./grammar/haxe.bnf
-java -jar grammar-kit.jar gen ./src/com/intellij/plugins/haxe/hxml/hxml.bnf
+#java -jar grammar-kit.jar gen ./src/common/com/intellij/plugins/haxe/hxml/hxml.bnf
```

Comment:
> Yes, we can fix the path, but the Grammar Kit 1.2.0 will generate code with a lot of incompatibilities

@winmain On my builds, generating the hxml classes by command line using GK 1.2.0 only had a couple of small changes required.  Check out the HxmlParserUpdatesForGrammarKit1.2.0 branch.

Also, what was the nature of the code differences you were seeing when regenerating the parser via the command line vs. IDEA?  Was it simply the variable name generation?  (I've figured that out.)